projects/core/src/lib/layers/layer-list/layer-list-element/layer-list-element.component.html

```diff
@@ -56,6 +56,10 @@
                 <mat-icon>file_download</mat-icon>
                 <span class="mat-body-2">Download Layer</span>
             </button>
+            <button mat-menu-item *ngIf="layer.layerType === 'vector'" (click)="showVectorDownload(layer)">
+                <mat-icon>file_download</mat-icon>
+                <span class="mat-body-2">Download Layer</span>
+            </button>
```

Comment:
You can remove the commented-out layer export button below, I guess.

Reply:
https://github.com/geo-engine/geoengine-ui/pull/722/commits/1e578a96e5bc69edf816f533eb70d4ceaa713c08